### ISSUE TITLE
`info` section of issue generated on netsplit is confusing

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#3513]
When servers detect a netsplit (with reql_admin) they generate a message in `rethinkdb.issues` and in that message there is a `info` section which is a little unclear, for example:

```python
'info': {'affected_servers': ['node_0'], 'server': 'node_1'}
```

This was generated by a server (`node_0`) that was isolated from the other server in its cluster (`node_`). My initial reading of this was that it is reversed from what is should be reporting. @chipotle made the suggestion that `affected_servers` should better be called `reporting_servers`, and that what is called `server` might better be called `affected_servers` (and be an array as well).

I don't have a list of all of the message types, but it might be worth re-considering hoisting these two fields out of `info` into the top level item, since every message should be reported by one or more servers and be about one or more servers.

Note: if this is changed then the `interface.detect_netsplit` test should be adjusted to match.

**Comment by danielmewes** [comment]
I agree that the current field names are really confusing.
I also like @chipotle's suggestion of renaming `affected_servers` to `reporting_servers`, assuming that also makes sense for the other issue types.

The `server` field for this issue should probably be called `inaccessible_server`?

Pinging @Tryneus @timmaxw

**Comment by Tryneus** [comment]
I agree with changing `affected_servers` to `reporting_servers` - keep in mind, this change should apply to any 'local' issue: `server_disconnected`, `server_ghost`, `log_write_error`, and `outdated_index`.  I think the name change is a straight improvement in all cases, though.

That said, I do not agree with changing `server` to `inaccessible_server` (if we change it at all, it should be `disconnected_server` for consistency).  The full issue structure should look like: 
```json
{
    "critical": True,
    "description": "[...]"
    "id": "2717a2c4-2571-571c-a54f-956153bc2a82",
    "info": {
        "reporting_servers": ["foo", "bar"],
        "server": "baz"
    },
    "type": "server_disconnected"
}
```

I think that taking the `type` into account, it is clear that the `server` field is talking about the disconnected server.  I don't feel too strongly about this, though.

Also, "inaccessible" appears a couple times in the `server_disconnected` issue description; it should be changed to "disconnected" (I missed this when making the terminology change).

**Comment by danielmewes** [comment]
I can live with leaving `server` as it is.
I think its meaning becomes clear in the context, assuming that we rename the other field to `reporting_servers`.

**Comment by danielmewes** [comment]
@Tryneus are you sure `affected_servers` is used in other local issues? Via grepping, I can only find it in `server_disconnected_issue_t::build_info_and_description()`.

**Comment by Tryneus** [comment]
@danielmewes, ah, my mistake, it looks like it was dropped from `server_ghost` and `outdated_index`.  In `log_write_error`, it is just `servers`, which is probably fine.

**Comment by timmaxw** [comment]
I would be in favor of changing `server` to `disconnected_server`. I think the clarity benefit outweighs the extra typing.

**Comment by danielmewes** [comment]
I'm fine with either option for `server`. My slight preference is with `disconnected_server` as well.

**Comment by Tryneus** [comment]
Settling this with:

* `affected_servers` -> `reporting servers`
* `server` -> `disconnected_server`
* Fix instances of `inaccessible` to `disconnected` in the issue description.

Working on this now.

**Comment by Tryneus** [comment]
The fix has been approved and merged to `next` in commit 4392628.  Review was 2457.  Will be in release 1.16.
